frontend/src/login/form/passcode.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+import { InputForm } from '../../components/input-form';
+
+const PasscodeInput = styled(InputForm)`
+  display: block;
+  margin-bottom: 12px;
+`;
+
+export type PasscodeFormProp = {
+  passcode?: string,
+  onSubmit?: (passcode: string) => void,
+
+  className?: string
+}
+
+export const PasscodeForm = ({
+  passcode,
+  onSubmit,
+
+  className,
+}: PasscodeFormProp) => {
+  function onInputHandler(text: string) {
+    if (text.length === 4) {
+      onSubmit?.(text);
+    }
+  }
+
+  if (passcode) {
+    onInputHandler(passcode);
+  }
```

Comment:
```suggestion
  useEffect(() => {
    if (passcode) {
      onInputHandler(passcode);
    }
  }, [passcode]);
```

이런 자잘하게 고칠 코드가 좀 있는데, 이번 PR에서 굳이 이런 코드들을 고치지 않을거면 해당 코멘트 접어주시면 감사하겠습니다

Reply:
이 부분은 useEffect를 사용해 처음 default value만 검사하도록 수정했습니다